storio-common/src/main/java/com/pushtorefresh/storio/util/QueryUtil.java

```diff
@@ -21,14 +21,14 @@ private QueryUtil() {
      * @return null if varargs array is null or empty or list of items from varargs
      */
     @Nullable
-    public static List<String> varargsToList(@Nullable Object... args) {
+    public static List<String> varargsToList(@Nullable Object[] args) {
         if (args == null || args.length == 0) {
             return null;
         } else {
             final List<String> list = new ArrayList<>(args.length);
 
-            for (int i = 0; i < args.length; i++) {
```

Comment:
This code will be faster on Android, unfortunately, Java compiler won't optimise this :(